tests/opentracing_instrumentation/test_tornado_http.py

```diff
@@ -73,3 +87,33 @@ def make_downstream_call():
         span.finish()
     assert response.code == 200
     assert response.body.decode('utf-8') == trace_id
+
+
+@pytest.mark.gen_test(run_sync=False)
+def test_http_fetch_with_interceptor(base_url, http_client, tornado_http_patch, tracer):
+
+    mock_interceptor = Mock(spec=OpentracingInterceptor)
+    Interceptors.append(mock_interceptor)
```

Comment:
It may be better to patch this interceptor in, so that the array is restored once this test exits. Otherwise it's creating a side effect that might mess up another test.

Reply:
this is a nice comment -- something that has definitely bitten me in the past 👍 

as an FYI -- same can be achieved by calling `clear()`, but clearly, that's easier to forget :)